papers/ring_vrf/ring_vrf.tex

```diff
@@ -82,15 +82,15 @@
 
 \input{overview}
 \input{background}
-\input{games}
+%\input{games} it is not complete so it should not be in the conference submission
 \input{uc_fun}
 \input{uc_model}
 
 \input{pedersen_vrf}
 \input{zk_cont}
 
-\input{ring_updates}
-\input{ring_hiding}
+%\input{ring_updates}
+%\input{ring_hiding} We cannot prove the UC security of our protocol with these options. Also, they are not formally described for a conference submission. We  should just say merkle tree or ring in the protocol description section
```

Comment:
I've kept ring_vrf.tex for the eprint and made submission specific files, ala https://github.com/w3f/ring-vrf/blob/master/papers/ring_vrf/usenix.tex